apps/nextjs-website/src/components/Form.tsx

```diff
@@ -0,0 +1,29 @@
+'use client';
+import MarkdownRenderer from './MarkdownRenderer';
+import { Form as FormRC } from '@react-components/components';
+import { FormProps } from '@react-components/types';
+import { FormSection } from '@/lib/fetch/types/PageSection';
+
+const makeFormProps = ({
+  title,
+  subtitle,
+  privacyText,
+  privacyLink,
+  privacyLinkText,
+  theme,
+  ctaButtons,
+  ...rest
+}: FormSection): FormProps => ({
+  ...rest,
+  title,
+  subtitle,
+  privacyText: MarkdownRenderer({ markdown: privacyText, variant: 'body2' }),
+  privacyLink,
+  privacyLinkText,
+  theme,
+  ctaButtons,
+});
```

Comment:
```suggestion
const makeFormProps = ({
  privacyText,
  ...rest
}: FormSection): FormProps => ({
  ...rest,
  privacyText: MarkdownRenderer({ markdown: privacyText, variant: 'body2' }),
  ...rest,
});
```

Removed everything unnecessary. If it's not modified, it can stay inside "rest"